sources/main.cpp

```diff
@@ -115,32 +117,36 @@ main(int argc, char* argv[]) {
     boost::timer timerSimu;
     context.execute();
 
-    // Traces must be re-initiliazed to append to current DynaflowLauncher log file as simulation had modified it
+    // Traces must be re-initialized to append to current DynaflowLauncher log file as simulation had modified it
     dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
     LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
     LOG(info) << " ============================================================ " << LOG_ENDL;
     LOG(info) << MESS(DFLEnded, context.basename(), timerGlobal.elapsed()) << LOG_ENDL;
 
     return EXIT_SUCCESS;
   } catch (DYN::Error& e) {
+    dfl::common::Log::init(options, outputDir, true);
```

Comment:
We cannot garrante that an exception doesn't occur here

Reply:
yes, but without this line the standard error in dynawo will never appear in the log, so I though it was a correct tradeoff?